js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts

```diff
@@ -452,6 +466,37 @@ export const AUDIO_MIME_TYPE =
 export const AUDIO_TRANSCRIPT =
   `${SemanticAttributePrefixes.audio}.${AudioAttributesPostfixes.transcript}` as const;
 
+/**
+ * The name of the agent.
+ */
+export const AGENT_NAME =
+  `${SemanticAttributePrefixes.agent}.${AgentPostfixes.name}` as const;
+/**
+ * The step number of the current agent in the execution graph.
+ */
+export const AGENT_STEP =
+  `${SemanticAttributePrefixes.agent}.${AgentPostfixes.step}` as const;
+/**
+ * The next agent to call
+ */
+export const AGENT_NEXT_AGENT =
+  `${SemanticAttributePrefixes.agent}.${AgentPostfixes.next_agent}` as const;
+/**
+ * The name of the node in the execution graph. Nodes are like agents but are for general purpose graphs.
+ */
+export const GRAPH_NODE_NAME =
+  `${SemanticAttributePrefixes.graph}.${GraphPostfixes.node_name}` as const;
```

Comment:
what do you think of leaving a comment here that these need to be somewhat unique, otherwise the graph is broken